app/views/certificates/i_belong/complete.html.erb

```diff
@@ -48,10 +48,16 @@
 
         <p class="govuk-body-l">Next steps for you</p>
         <p class="govuk-body">
-          This is just the start of your journey! Now you’ve started implementing strategies to support girls into computer science at key stage 4 and beyond, it’s important to continue doing so, embedding and evaluating as you go along. Evidence your great work through the
+          This is just the start of your journey!
+        </p>
+        <p class="govuk-body">
+          Now you’ve started implementing strategies to support girls into computer science from primary, during the transition, in secondary and beyond, it’s important to continue doing so, embedding and evaluation as you go along. Evidence your great work through the
           <%= link_to 'Computing Quality Framework.', 'https://computingqualityframework.org/' %>
         </p>
-        <p class="govuk-body">Help to widen impact through your network by sharing your knowledge and practice. If you haven’t already, don’t forget to claim your I Belong champion digital badge and check the <%= link_to 'information pack', 'https://static.teachcomputing.org/IBelong_InfoPack.pdf' %> for ideas and support on how you can do this.</p>
+        <p class="govuk-body">
+          Help to widen impact through your network by sharing your knowledge and practice – you can check out the I Belong Champion <%= link_to 'information pack', 'https://static.teachcomputing.org/IBelong_InfoPack.pdf' %> for ideas and support on how you can do this.
```

Comment:
Should we move the Info Pack link to a external link helper method?

And do the computing quality framework one at the same time?

Reply:
Not sure why I didn't do this to start with, we already have a helper method for CQF. I will move the info pack too.